rosserial_tivac/src/ros_lib/startup_gcc.c

```diff
@@ -0,0 +1,490 @@
+/*
+ * Copyright (c) 2015, Robosavvy Ltd.
```

Comment:
This looks like a TI-supplied file. Are you sure you're clear to assert copyright over it?